nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/test/java/org/apache/nifi/controller/repository/TestStandardProcessSession.java

```diff
@@ -243,6 +243,16 @@ public void process(OutputStream out) throws IOException {
 
         flowFile = session.get();
         assertNotNull(flowFile);
+        assertNotNull(flowFile.getAttribute(StandardProcessSession.ROLLBACK_COUNT_ATTR_NAME));
+        assertEquals("1", flowFile.getAttribute(StandardProcessSession.ROLLBACK_COUNT_ATTR_NAME));
+        session.remove(flowFile);
+        session.rollback();
+        assertEquals(1, contentRepo.getExistingClaims().size());
+
+        flowFile = session.get();
+        assertNotNull(flowFile);
+        assertNotNull(flowFile.getAttribute(StandardProcessSession.ROLLBACK_COUNT_ATTR_NAME));
```

Comment:
not specific to this line but we should probably have details about a flow files's rollback status/count/etc.. as part of the FlowFileRecord object rather than the FlowFile class attributes.  The idea I was hinting about was more about making it possible when viewing information about a queue to see flow files involved in a rollback recently to aid debugging.  Putting it as attributes, and ones which would not be persisted as is the case here (by design in a rollback), heads in a different direction.